docs/docs/preprocessing-external-images.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Preprocessing External Images
+---
+
+Gatsby allows powerful image processing features through using the [`Sharp`](https://github.com/lovell/sharp/) library to automatically process images to be performant and have features like lazy-loading. That said, this only works if the image is a file node in the GraphQL layer.
+
+That said, if you want the same ecosystem for files that are hosted online and not located in your repo, [`gatsby-source-filesystem`](/packages/gatsby-source-filesystem/) has an API called `createRemoteFileNode` to solve this.
```

Comment:
Two times `That said,` is not that nice.

Reply:
Resolved. Took off the 2nd "That said"